app/Http/Controllers/UserController.php

```diff
@@ -109,17 +109,24 @@ public function requestPasswordReset(Request $request)
         'email' => 'required|email',
     ]);
 
-    return $this->userService->requestPasswordReset($data['email']);
+    return $this->userService->requestPasswordReset($data);
 }
 
 public function resetPassword(Request $request)
 {
     $data = $request->validate([
+        'email' => 'required|email',
         'token' => 'required|string',
-        'newPassword' => 'required|string|max:255',
+        'newPassword' => 'required|string|min:6',
+        'newPassword_confirmation' => 'required|string|same:newPassword',
```

Comment:
you don't need to have two fields for password conformation
'newPassword' => 'required|string|confirmed|min:6', -> with this confirmed validation it will automaticlly require you to write the password twice